src/zope/app/authentication/browser/schemasearch.py

```diff
@@ -94,7 +95,7 @@ def render(self, name):
         html.append('<div class="row">')
         html.append('  <div class="field">')
         html.append('    <input type="submit" name="%s" value="%s" />'
-                     % (name+'.search',
+                    % (name+'.search',
                         translate(search_label, context=self.request)))
```

Comment:
I'm mildly surprised flake8 doesn't insist `translate(...)` here be de-dented to align with `name` above.

Reply:
I fixed it and added spaces around the `+`, where I was surprised that this was not caught.